hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/impl/TestContainerPersistence.java

```diff
@@ -810,7 +811,7 @@ public void testPutBlockWithInvalidBCSId(ContainerTestVersionInfo versionInfo)
           getBlock(container, blockID1);
       Assertions.fail("Expected exception not thrown");
     } catch (StorageContainerException sce) {
-      Assertions.assertTrue(sce.getResult() == UNKNOWN_BCSID);
+      assertSame(UNKNOWN_BCSID, sce.getResult());
```

Comment:
Should this be `assertEquals`?

Reply:
`assertSame` for objects verifies reference equality, like `assertTrue( == )`.

Equality of enum values can be checked both with `equals` and `==`, since each value is a singleton.